index.js

```diff
@@ -36,6 +37,11 @@ class Dalai {
     await fs.promises.mkdir(resolvedPath, { recursive: true }).catch((e) => { })
 
     for(let file of files) {
+      if (fs.existsSync(path.resolve(resolvedPath, file))) {
+        console.log(`Skip file download, it already exists: ${file}`)
+        continue;
+      }
+
```

Comment:
Very useful.  The 6th parameter file of the 65B download kept failing for me.  I had to pull it from a torrent to get it to work, then had to do the quantization manually.   This and the quantization file check below improve error recover a lot.